LICENSE-README.md

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2023 See the [README.md](README.md) at the bottom for authors/contributors.
+
+This software is a digital implementation of the "FüSim MANV"/"FueSim MANV" (Führungssimulation Massenanfall von Verletzen), a German simulation system for training emergency medical services leadership personnel on how to manage Mass Casualty Incidents.
+
+The software is licensed with the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+Exceptions apply to image files in the frontend/src/assets folder, which may be covered by and therefore distributed under different licenses.
+The individual license for each image can be found in [frontend/src/assets/LICENSES-IMAGES.md](frontend/src/assets/LICENSES-IMAGES.md). Files added to this project may only use this exception if these files serve the only purpose of being displayed and will only be parsed for the purpose of displaying them.
+Another exception exists for datasets. Datasets are files that include no new features and just different compositions allowed through the source code given template class. Datasets must always be in human readable formats (for example csv, JSON, txt). Datasets license must be mentioned in a LICENSE-DATASET.md (or LICENSES-DATASETS.md) file in the folder where the dataset is placed, stating which file the license applies to and give information about the license and author. These datasets are not allowed to create any new features or enhance the software in any way, as to create elements that can be part of an export and are importable into the distributed source code (this software) under the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```

Comment:
Add blank line (paragraph separator) before the dataset part.

Also, I propose some minor changes for readability.
```suggestion
Another exception to the default licence exists for datasets. Datasets are files that include no new features and just contain different compositions allowed through template classes given in the source code. Datasets must always be in human readable formats (for example csv, JSON, txt). Datasets license must be mentioned in a LICENSE-DATASET.md (or LICENSES-DATASETS.md) file in the respective folder where the dataset is placed. The licence files must state explicitly which files they refer to, who the authors are and which license applies. Datasets are not allowed to create any new features or enhance the software in any way, as to create elements that can be part of an export and are importable into the distributed source code (this software) under the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```